Gemfile

```diff
@@ -1,7 +1,12 @@
+# frozen_string_literal: true
+
 source "https://rubygems.org"
 
 # Specify your gem's dependencies in aca_entities.gemspec
 gemspec
 
 gem "rake", "~> 12.0"
 gem "rspec", "~> 3.0"
+
+gem "rubocop"
```

Comment:
Wrap these in 

group :development, :test do
  gem "rubocop"
  gem "yarn"
end